src/e2e/java/teammates/e2e/util/ReportAnalysis.java

```diff
@@ -0,0 +1,62 @@
+package teammates.e2e.util;
+
+/**
+ * Report analysis class for L&P tests.
+ */
+public class ReportAnalysis {
+
+    private double meanResTime;
+    private double pct1ResTime; // 90th percentile
+    private double throughput;
+    private int errorCount;
+    private int sampleCount;
+    private String analysisFeedback = "";
+    private String testName;
+
+    /**
+     * Generate a feedback message based on test analysis with the provided threshold.
+     */
+    public void generateAnalysisFeedback(double errorRateLimit, double meanResTimeLimit) {
+        checkErrorLimit(errorRateLimit);
+        checkMeanResTimeLimit(meanResTimeLimit);
+        if ("".equals(analysisFeedback)) {
+            System.out.println("You have successfully passed the default profiling threshold for " + testName);
```

Comment:
Do we need this?

Btw this method actually solves half of [this TODO](https://github.com/TEAMMATES/teammates/pull/9652/files#diff-6509e6ed4f2a3ea02c0d46c61b1ff10fR372).

Reply:
I was thinking we should inform the user about it to show that we checked with our internal threshold values?

Reply:
That should be implied, right? We have the assertion error in place, if it doesn't satisfy the thresholds